src/agat/agat_sp_merge_annotations/script.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+## VIASH START
+## VIASH END
+
+# Convert a list of file names to multiple -gff arguments
+input_files=""
+IFS=";" read -ra file_names <<< "$par_gff"
+for file in "${file_names[@]}"; do
+    input_files+="--gff $file "
+done
+unset IFS
+
+# run agat_sp_merge_annotations
+agat_sp_merge_annotations.pl \
+  $input_files \
+  -o "$par_output" \
+  ${par_config:+--config "${par_config}"}
```

Comment:
Could you test if the following works?
A couple of things here:

1. If you set an environment variable as part of a composite command instead of on a separate line, it is only applied for the commands on that line (here IFS is only used by `read`, so no need to use `unset` afterwards)
2. The `${array[@]}` syntax can be used to output an array which is seprated by spaces (space is the default separator, this can be adjusted)

```suggestion
IFS=";" read -ra file_names <<< "$par_gff"

# run agat_sp_merge_annotations
agat_sp_merge_annotations.pl \
  $input_files \
  -o "$par_output" \
  ${file_names[@]} \
  ${par_config:+--config "${par_config}"}
```

Reply:
it does not, the input should be in this format: --gff input1.gff --gff input2.gff